exchanges/lbank/lbank_types.go

```diff
@@ -0,0 +1,243 @@
+package lbank
+
+import (
+	"encoding/json"
+)
+
+// Ticker stores the ticker price data for a currency pair
+type Ticker struct {
+	Change   float64 `json:"change"`
+	High     float64 `json:"high"`
+	Latest   float64 `json:"latest"`
+	Low      float64 `json:"low"`
+	Turnover float64 `json:"turnover"`
+	Volume   float64 `json:"vol"`
+}
+
+// TickerResponse stores the ticker price data and timestamp for a currency pair
+type TickerResponse struct {
+	Symbol    string `json:"symbol"`
+	Timestamp int64  `json:"timestamp"`
+	Ticker    Ticker `json:"ticker"`
+}
+
+// MarketDepthResponse stores arrays for asks, bids and a timestamp for a currecy pair
+type MarketDepthResponse struct {
+	Asks      [][]float64 `json:"asks"`
+	Bids      [][]float64 `json:"bids"`
+	Timestamp int64       `json:"timestamp"`
+}
+
+// TradeResponse stores date_ms, amount, price, type, tid for a currency pair
+type TradeResponse struct {
+	DateMS int64   `json:"date_ms"`
+	Amount float64 `json:"amount"`
+	Price  float64 `json:"price"`
+	Type   string  `json:"type"`
+	TID    string  `json:"tid"`
+}
+
+// KlineResponse stores kline info for given currency exchange
+type KlineResponse struct {
+	TimeStamp     int64   `json:"timestamp"`
+	OpenPrice     float64 `json:"openprice"`
+	HigestPrice   float64 `json:"highestprice"`
+	LowestPrice   float64 `json:"lowestprice"`
+	ClosePrice    float64 `json:"closeprice"`
+	TradingVolume float64 `json:"tradingvolume"`
+}
+
+// InfoResponse stores info
+type InfoResponse struct {
```

Comment:
This struct is incorrect based on the data lbank return

The format is:
```js
 {
 	"result": "true",
 	"info": {
 		"freeze": {
 			"currency": "value"
 		},
 		"asset": {
 			"currency": "value"
 		},
 		"free": {
 			"currency": "value"
 		}
 	}
 }
```

which would be 
```go
struct {
    string
    struct { 
        map[string]string
        map[string]string
        map[string]string
    }
}
```